files/dev-sec.conf

```diff
@@ -0,0 +1,8 @@
+install cramfs /bin/true
```

Comment:
@bitvijays as you want to split it to the smaller PRs (very good idea!), maybe lets start with this thing